tests/conftest.py

```diff
@@ -85,7 +85,7 @@
 
 
 class CaptureQueriesContext(BaseCaptureQueriesContext):
-    IGNORED_QUERIES = settings.PATTERNS_IGNORED_IN_QUERY_CAPTURES
+    IGNORED_QUERIES = "" #settings.PATTERNS_IGNORED_IN_QUERY_CAPTURES
```

Comment:
Is it expected?

Reply:
For some reason this causes the tests not to run, will undo this change before merging

Reply:
So the error I get when I leave this in is:
```
ImportError while loading conftest '/Users/ewanvalentine/Development/saleor/tests/conftest.py'.
tests/conftest.py:87: in <module>
    class CaptureQueriesContext(BaseCaptureQueriesContext):
tests/conftest.py:88: in CaptureQueriesContext
    IGNORED_QUERIES = settings.PATTERNS_IGNORED_IN_QUERY_CAPTURES
venv/lib/python3.8/site-packages/django/conf/__init__.py:80: in __getattr__
    val = getattr(self._wrapped, name)
E   AttributeError: 'Settings' object has no attribute 'PATTERNS_IGNORED_IN_QUERY_CAPTURES'
```

I also get an error with the git commit hook, which might be an issue with my local python installation perhaps, but I can't commit without doing no-verify unfortunately: 
```
    urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1108)>
```